optimum/habana/transformers/trainer.py

```diff
@@ -1477,7 +1477,7 @@ def evaluation_loop(
             if not self.already_wrapped_for_hpu_graphs:
                 from habana_frameworks.torch.hpu import wrap_in_hpu_graph
 
-                model = wrap_in_hpu_graph(model)
+                model = wrap_in_hpu_graph(model, disable_tensor_cache=False, max_graphs=None)
```

Comment:
```suggestion
                model = wrap_in_hpu_graph(model, disable_tensor_cache=args.disable_tensor_cache_hpu_graphs, max_graphs=args.max_hpu_graphs)
```
I guess we want to use the new training args you defined.